solr/core/src/test/org/apache/solr/cloud/TestZkChroot.java

```diff
@@ -27,8 +27,10 @@
 import org.apache.solr.core.CoreContainer;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
+// TODO: this class tries to test Zookeeper using Solr abstractions, but ZK implies the code is running in cloud mode. It doesn't work.
```

Comment:
Perhaps MiniSolrCloud should have a `.withChroot` option which can be asserted somewhere? What about deleting this test and create a separate followup JIRA "Add zk chroot test" which can be fixed as a followup?

Reply:
I agree (on the followup option).
I tried to see if there's a way to do the chroot on the mini cluster or elsewhere but nothing obvious came after an hour or two of hacking, that's why I suggest to leave it for later.